app/assets/stylesheets/reskin/moneyhelper_variables.scss

```diff
@@ -0,0 +1,86 @@
+// -- Colours -----------------------------------------
+$white: #ffffff;
+
+$text_secondary: #000B3B;
+
+$input_border: #60637E;
+
+$blue_primary: #0F19A0;
+
+$magenta_primary: #C82A87;
+$magenta_dark: #AE0060;
+$magenta_darker: #9C0052;
+$button_active_bg: #9C0052;
+$light_grayish_magenta: #eeebee;
+
+$yellow_secondary: #F0F05A;
+
+$teal_secondary: #00788F;
+
+$red_error: #D00000;
+$red_error_background: #ffeef0;
+
+$support_focus: #8200D1;
+$support_outline: #9DA1CA;
+
+$link_visited: #792B9E;
+
+$light_grey: #F3F3FA;
+$light_grey_2: #F3F1F3;
+// ----------------------------------------------------
+
+// -- Dimensions --------------------------------------
+$border_radius: 4px;
+$mobile_width: 768px;
+// ----------------------------------------------------
+
+// -- Fonts -------------------------------------------
+@font-face{
+    font-family: 'Roobert-Regular';
+    src: font-url('Roobert-Regular.woff2') format('woff2'),
+         font-url('Roobert-Regular.woff') format('woff');
+}
+@font-face{
+    font-family: 'Roobert-bold';
+    src: font-url('Roobert-Bold.woff2') format('woff2'),
+         font-url('Roobert-Bold.woff') format('woff');
+}
+
+@font-face{
+    font-family: 'Roobert-SemiBold';
+    src: font-url('Roobert-SemiBold.woff2') format('woff2'),
+         font-url('Roobert-SemiBold.woff') format('woff');
+}
```

Comment:
If we are going to load all the variations (weight, italic, format) of the font (see earlier comment) then we should include them here in a `@font-face` declaration, otherwise they will never get used. It will also ensure better quality of type rendering.

Reply:
As a future piece of work we should also define the `@font-face` rules in Dough and delete the rules in this file.